HotelBooking/HotelBooking.DataAccessLayer/Migrations/20241202170917_AddPasswordHashColumn.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace HotelBooking.DataAccessLayer.Migrations
+{
+    /// <inheritdoc />
+    public partial class AddPasswordHashColumn : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<string>(
+                name: "PasswordHash",
+                table: "Guests",
+                type: "nvarchar(max)",
+                nullable: true);
```

Comment:
hasło nie powinno być nullable.

Reply:
ten nvarchar(max) też jest dyskusyjne, ale to jest hash więc ciężko przewidzieć ile mogłoby zając miejsca więc może zostać. Zwracam tylko uwagę żeby zwracać uwagę na takie kwestie :)